src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ArgCoercion.txt

```diff
@@ -16,11 +16,9 @@
 "123456"
 
 >> Concatenate(-1, ["456", "abc"] )
-/*InterpreterRunner*/ #Skip
 [{Result:"-1456"},{Result:"-1abc"}]
 
 >> Concatenate([-1, 123], ["456", "abc"] )
-/*InterpreterRunner*/ #Skip
 [{Result:"-1456"},{Result:"123abc"}]
 
```

Comment:
We should have many more tests for the "zipping" scenarios:
- Scalar+Table
  - Simple case (what we have in line 18)
  - Empty table
  - Null table (e.g., `If(1<0,["txt"])`)
  - Error table (e.g., `If(1/0<2,["txt"])`)
  - Table with error values (e.g., `["hello", Mid("great", -1), "world"]`)
  - Table with error records (e.g., `Table({a:"one"},If(1/0<2,{a:"two"}),{a:"three"})`)
  - Table with null values (e.g., `Table({a:"one"},If(1<0,{a:"two"}),{a:"three"))`)
  - Null scalar + valid table
  - Null scalar + table with errors
- Table+Scalar (not there)
  - Same cases listed above, opposite order
- Table+Table
  - Simple case (what we have in line 21)
  - Error/null tables
  - Tables with error / null values
  - Tables with different number of elements (e.g., `Concatenate(["one", "two"], [1, 2, 3, 4])`)

Reply:
Ok, I added all those tests and they are passing!